cmd/sup/main.go

```diff
@@ -99,6 +100,12 @@ func parseArgs(conf *sup.Supfile) (*sup.Network, []*sup.Command, error) {
 	// Add default env variable with current network
 	network.Env["SUP_NETWORK"] = args[0]
 
+	// Add default nonce
+	network.Env["SUP_NONCE"] = time.Now().Format("2006-01-02T15:04:05")
```

Comment:
I think this PR makes sense -- but what time format should we actually go with?

* Unix timestamp..?
* "YYYY-MM-DD-HH:MM:SS" ? I prefer this over whats in the PR, and keep it UTC